### PR TITLE
Updated springboard alerts from Xcode 12

### PR DESCRIPTION
**Pull request by IlyaBausovAkvelon**

### CODE REVIEW

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-el.dataset/alerts.json [hunk 1]
```diff
@@ -499,29 +500,61 @@
...
+      "Να επιτρέπεται",
+      "Να επιτραπεί"
     ],
     "shouldAccept": true
   },
   {
     "title": "Αίτημα από «%@» για εύρεση και σύνδεση σε συσκευές στο τοπικό σας δίκτυο.",
+    "buttons": [
+      "OK",
+      "ΟΚ"
```
Comment (AndreyMaslennikov) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-el.dataset/alerts.json:528]: Do we really need to keep duplicated buttons?

**Review by JoeSSS — APPROVED** [review]

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json [hunk 2]
```diff
@@ -449,5 +449,63 @@
...
+      "Permitir"
+    ],
+    "shouldAccept": true
+  },
+  {
+    "title": "¿Activar dictado?",
+    "buttons": [
+      "Ahora no"
+    ],
+    "shouldAccept": true
```
Comment (AndreyMaslennikov) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json:481]: We should accept this alert, but "Ahora no" translates as "Not now"

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json [hunk 3]
```diff
@@ -449,5 +449,63 @@
...
+  },
+  {
+    "title": "Activar dictado",
+    "buttons": [
+      "Ahora no"
+    ],
+    "shouldAccept": true
```
Comment (AndreyMaslennikov) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json:488]: The same as above

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-pt_BR.dataset/alerts.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "title": "“%1$@” Deseja Ter Acesso às Contas do %2$@",
```
Comment (AndreyMaslennikov) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-pt_BR.dataset/alerts.json:3]: What does `%1$@` and `%2$@` mean?
AFAIK, the app name in the title should be replaced with `%@`. Are these two parameters?

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-ru.dataset/alerts.json [hunk 5]
```diff
@@ -571,7 +571,36 @@
   {
     "title": "Приложение «%@» запрашивает разрешение на поиск устройств в Вашей локальной сети и подключение к ним.",
     "buttons": [
-      "OK"
+      "OK",
+      "ОК"
```
Comment (AndreyMaslennikov) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-ru.dataset/alerts.json:575]: Same as above for duplicated buttons. But it looks like the first "OK" is typed in latin letters, and the second "ОК" - in Cyrillic.🙂
Could you please confirm?

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-el.dataset/alerts.json (hunk 1 above)
Comment (AndreyMaslennikov) [reply Server/Resources.xcassets/springboard-alerts/springboard-alerts-el.dataset/alerts.json:528]: Looks like these to "OK" were typed in different letters (Latin and Cyrillic probably)
Could you please confirm?

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json (hunk 2 above)
Comment (AndreyMaslennikov) [reply Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json:481]: We should fix this button at least.
For other comments we need just a small confirmation or explanation.

**Review by AndreyMaslennikov — CHANGES_REQUESTED** [review]

**Review by AndreyMaslennikov — APPROVED** [review]

**Pull request merged**